ui/apps/platform/src/Containers/MainPage/AnnouncementBanner.tsx

```diff
@@ -1,35 +1,49 @@
 import React, { ReactElement, useEffect, useState } from 'react';
 import { AlertVariant, Banner, Button } from '@patternfly/react-core';
 
+import { fetchDatabaseStatus } from 'services/DatabaseService';
+
 const ANNOUNCEMENT_BANNER_KEY = 'postgresAnnouncementBannerDismissed';
 
 function AnnouncementBanner(): ReactElement | null {
     const [isDisplayed, setIsDisplayed] = useState(false);
+    const [databaseType, setDatabaseType] = useState('');
 
     useEffect(() => {
         const localStorageValue = localStorage.getItem(ANNOUNCEMENT_BANNER_KEY);
         const isBannerDismissed = localStorageValue
             ? Boolean(JSON.parse(localStorageValue))
             : false;
         setIsDisplayed(!isBannerDismissed);
+
+        if (!isBannerDismissed) {
+            fetchDatabaseStatus()
+                .then((response) => {
+                    setDatabaseType(response?.databaseType || '');
+                })
+                .catch(() => {
+                    setDatabaseType('');
+                });
+        }
     }, []);
 
     function handleDismissClick() {
         localStorage.setItem(ANNOUNCEMENT_BANNER_KEY, JSON.stringify(true));
         setIsDisplayed(false);
     }
 
-    if (isDisplayed) {
+    if (isDisplayed && databaseType !== 'PostgresDB') {
         return (
             <Banner
                 className="pf-u-display-flex pf-u-justify-content-center pf-u-align-items-center"
                 isSticky
                 variant={AlertVariant.info}
+                style={{ whiteSpace: 'normal' }}
             >
                 <span className="pf-u-text-align-center">
-                    The next version of this product will be version 4.0.0. Central will be using
-                    Postgres for its data store starting in v4.0.0. You must backup your database
-                    before upgrading to 4.0.0.
+                    Red Hat Advanced Cluster Security plans to change its database to PostgreSQL in
```

Comment:
Nitpicking: "Red Hat Advanced Cluster Security" is valid for RHACS_BRANDING. Could you please use branded product name so that "StackRox" shows up for community builds?

Reply:
This would be different than our current practice, where the UI does not change for anything at runtime. Only the logo changes at build time, based on ACS vs. opensource.

Reply:
I believe that info is available at build time. Legitimately hardcoded here:
https://github.com/stackrox/stackrox/blob/baada33c57ee29ae0527d470bc31be3f0143cd68/ui/apps/platform/src/constants/productBranding.ts#L33